README.md

```diff
@@ -44,6 +76,15 @@ You can also get in touch directly: gnark@consensys.net
 
 [Release Notes](CHANGELOG.md)
 
+## Audits
+
```

Comment:
we may also add the one for worldcoin: https://leastauthority.com/wp-content/uploads/2023/08/Worldcoin_Groth16_Verifier_in_EVM_Smart_Contract_Final_Audit_Report.pdf

Reply:
Added